frontend/packages/schema-editor/src/components/SchemaInspector/hooks/useKindOption.ts

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { ObjectKind } from '@altinn/schema-model';
+
+type kindOption = {
+  kind: ObjectKind;
+  label: string;
+};
+
+export const useKindOptions = (): kindOption[] => {
```

Comment:
Vi bruker stor forbokstav på typer.
```suggestion
type KindOption = {
  kind: ObjectKind;
  label: string;
};

export const useKindOptions = (): KindOption[] => {
```

Reply:
den er fikset.